snap/hooks/install

```diff
@@ -189,7 +189,7 @@ snapctl stop "$SNAP_NAME.postgres"
 # off.
```

Comment:
the comment in the line above this is wrong now

Reply:
Just changed the comment about disabling services:

```
 # finally, disable and turn off non-default services
-# by default, we want the support-*, device-*, and redis services
+# by default, we want the support-*, device-*, and mongo services

```